### PR TITLE
[es] add popular words

### PR DESCRIPTION
**Pull request by tiff**
Popular words that users from spellboy.com added to their dictionary.

### CODE REVIEW

File: languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/hunspell/spelling.txt [hunk 1]
```diff
@@ -151,3 +152,50 @@ Xbox
 Twitch
 iOS
 Alibaba
+concientizar
```
Comment (tiff) [inline languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/hunspell/spelling.txt]: http://www.spanishdict.com/translate/concientizar

File: languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/hunspell/spelling.txt [hunk 2]
```diff
@@ -151,3 +152,50 @@ Xbox
...
+UPS
+abuelita
```
Comment (tiff) [inline languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/hunspell/spelling.txt]: http://www.spanishdict.com/translate/abuelita

File: languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/hunspell/spelling.txt [hunk 3]
```diff
@@ -151,3 +152,50 @@ Xbox
...
+casita
```
Comment (tiff) [inline languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/hunspell/spelling.txt]: http://www.spanishdict.com/translate/casita

do we need `casitas` as well?

File: languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/hunspell/spelling.txt [hunk 4]
```diff
@@ -151,3 +152,50 @@ Xbox
...
+City
+Center
+River
+New
```
Comment (tiff) [inline languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/hunspell/spelling.txt]: English but popular Words. e.g. New York

File: languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/hunspell/spelling.txt [hunk 5]
```diff
@@ -151,3 +152,50 @@ Xbox
...
+mami
+papi
+Smith
+Hey
+AM
+PM
```
Comment (tiff) [inline languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/hunspell/spelling.txt]: https://spanish.stackexchange.com/questions/888/use-of-am-pm-in-time

not sure if this should be upper and/or down cased

File: languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/hunspell/spelling.txt [hunk 6]
```diff
@@ -151,3 +152,50 @@ Xbox
...
+Perez
+Tom
+Ivan
+Isaías
+Lucy
+Omar
+Luke
+Karla
+Jacob
+personita
```
Comment (tiff) [inline languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/hunspell/spelling.txt]: english: "little person"
do we also need `personitas`?

File: languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/hunspell/spelling.txt [hunk 7]
```diff
@@ -151,3 +152,50 @@ Xbox
...
+contáctanos
+desabasto
+pedacito
```
Comment (tiff) [inline languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/hunspell/spelling.txt]: http://www.spanishdict.com/translate/pedacito

File: languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/hunspell/spelling.txt [hunk 8]
```diff
@@ -151,3 +152,50 @@ Xbox
...
+Jones
+Gonzalo
+cajita
+cajitas
+EEUU
+EU
+AMLO
```
Comment (tiff) [inline languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/hunspell/spelling.txt]: disambiguation of new mexican persident: https://en.wikipedia.org/wiki/Andr%C3%A9s_Manuel_L%C3%B3pez_Obrador

File: languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/hunspell/spelling.txt [hunk 9]
```diff
@@ -151,3 +152,50 @@ Xbox
...
+CDMX
```
Comment (tiff) [inline languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/hunspell/spelling.txt]: disambiguation of ciudad de mexico

File: languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/hunspell/spelling.txt [hunk 10]
```diff
@@ -151,3 +152,50 @@ Xbox
...
+faltante
```
Comment (tiff) [inline languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/hunspell/spelling.txt]: http://www.spanishdict.com/translate/faltante

File: languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/hunspell/spelling.txt [hunk 11]
```diff
@@ -151,3 +152,50 @@ Xbox
...
+jovencito
```
Comment (tiff) [inline languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/hunspell/spelling.txt]: http://www.spanishdict.com/translate/jovencito

File: languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/hunspell/spelling.txt [hunk 12]
```diff
@@ -151,3 +152,50 @@ Xbox
...
+regalito
```
Comment (tiff) [inline languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/hunspell/spelling.txt]: http://www.spanishdict.com/translate/regalito

do we need plural here?

File: languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/hunspell/spelling.txt [hunk 13]
```diff
@@ -151,3 +152,50 @@ Xbox
...
+derrepente
```
Comment (tiff) [inline languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/hunspell/spelling.txt]: not sure about this
http://www.spanishdict.com/translate/derrepente?t=1

File: languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/hunspell/spelling.txt [hunk 14]
```diff
@@ -151,3 +152,50 @@ Xbox
...
+religionista
+religionistas
```
Comment (tiff) [inline languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/hunspell/spelling.txt]: http://www.spanishdict.com/translate/religionista

**Comment by tiff** [comment]
@jmartorell hey! any chance you could have a look at this? I would be happy to get your feedback

**Comment by TiagoSantos81** [comment]
@tiff Looks good but I will remove English words that are used as nouns in phrases(New, City, AM, etc.) and non-standard Spanish words (e.g. religionista, derrepente, etc.).

**Comment by tiff** [comment]
@TiagoSantos81 thank you!

**Pull request merged**